cms/test_utils/project/customuserapp/models.py

```diff
@@ -39,7 +40,7 @@ class User(AbstractBaseUser, PermissionsMixin):
         help_text=_('Required. 300 characters or fewer. Letters, numbers and '
                     '@/./+/-/_ characters'),
         validators=[
-            validators.RegexValidator(re.compile('^[\w.@+-]+$'), _('Enter a valid username.'), 'invalid')
```

Comment:
@marksweb Is this regex change intended?

Reply:
@Aiky30 I don't know if a tool that auto-fixes flake8 errors, though I'd love one. isort auto-fixes the imports, but flake8 errors need manually resolving

Reply:
@marksweb I've just tested the revised regex and it fails for email@here.com, try it yourself: https://www.freeformatter.com/regex-tester.html

Reply:
This exposes a missing unit test too

Reply:
That's a regex in the test utilities, right? I'd assume someone at some time just copied it there. It's not part of the code base.